docs/jobs.md

```diff
@@ -39,11 +39,21 @@ import socket
 def add(x, y):
     job = get_current_job()
     job.meta['handled_by'] = socket.gethostname()
-    job.save()
+    job.update_meta()
+    
+    # do more work
+    time.sleep(1)
     return x + y
 {% endhighlight %}
 
-
+<div class="warning">
+    <img style="float: right; margin-right: -60px; margin-top: -38px" src="{{site.baseurl}}img/warning.png" />
+    <strong>Warning!</strong>
+    <p>
+The `.update_meta()` was introduced in RQ 0.7.2. You can use the `.save()` method instead 
+for RQ <= 0.7.1. For that case you also have to set the job `ttl` greater than the job `timeout`, otherwise the job may dissappear from the queue during execution.
```

Comment:
Hmm... My idea was to refer somehow to the #811. If I `get_current_job` and then `save` it during execution, `cleanup` sets back expiration time to the Redis key. If the job lasts more than `ttl` since that moment, the Redis key expires. So if I don't want the key to expire, I have to set `ttl > timeout` (i.e. an expected max job duration).
@selwin, what do you think, should we mentioned it here?